HaroldAdviser/Models/WarningModel.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace HaroldAdviser.Models
```

Comment:
HaroldAdviser.Models namespace is also used in HaroldAdviser.Models subproject. It may cause some problems in future. Maybe you should rename it.

Reply:
done

Reply:
done